src/llvmo/llvmoExpose.cc

```diff
@@ -642,23 +632,38 @@ CL_DEFMETHOD core::T_sp TargetMachine_O::addPassesToEmitFileAndRunPassManager(Pa
   } else {
     SIMPLE_ERROR("Illegal file type {} for addPassesToEmitFileAndRunPassManager", _rep_(dwo_stream));
   }
-  if (this->wrappedPtr()->addPassesToEmitFile(*passManager->wrappedPtr(), *ostream.get_stream(), dwo_ostream.get_stream(), FileType, true, nullptr)) {
+
+  llvm::legacy::PassManager PM;
+
+  llvm::LLVMTargetMachine* LTM = dynamic_cast<llvm::LLVMTargetMachine*>(TM);
+  if (LTM) {
+    llvm::TargetPassConfig* TPC = LTM->createPassConfig(PM);
+    TPC->setEnableTailMerge(false);
```

Comment:
might be good to mention why we're doing this - iirc drmeister disabled tail merges so that debug info is more accurate. otherwise it just seems kind of random to do it here

Reply:
Add comment for that.